install.sh

```diff
@@ -0,0 +1,285 @@
+#!/bin/bash
+# OS Defaults
+OS=$(uname -s) # Gather OS Name
+USER=$(whoami) # Gather USER Name
+
+#OntoUML-Vp-Plugin Defaults
+repo_name="ontouml-vp-plugin"
+repo_url="https://github.com/propilideno/ontouml-vp-plugin/archive/refs/heads/master.zip"
+ontouml_plugin_path="ontouml-vp-plugin-0.5.3"
+
+# =========== Visual Paradigm Defaults ===========
+#App Default Path
+VISUAL_PARADIGM_APP_DIR_WINDOWS="C:\\Program Files\\Visual Paradigm CE 17.0\\" #Windows
+VISUAL_PARADIGM_APP_DIR_MAC="/Applications/Visual Paradigm.app/Contents/Resources/app/" #MacOS
+VISUAL_PARADIGM_APP_DIR_LINUX="/home/$USER/Visual_Paradigm_17.0/" #Linux
+#Plugin Default Path
+VISUAL_PARADIGM_PLUGIN_DIR_WINDOWS="C:\\Users\\$USER\\AppData\\Roaming\\VisualParadigm\\plugins\\" #Windows
+VISUAL_PARADIGM_PLUGIN_DIR_MAC="/Users/$USER/Library/Application Support/VisualParadigm/plugins/" #MacOS
+VISUAL_PARADIGM_PLUGIN_DIR_LINUX="/home/$USER/.config/VisualParadigm/plugins/" #Linux
+
+function read_App_Path(){
+    while true; do
+        echo "Visual Paradigm Path: $1"
+        read -p "Confirm (y/n)?: " choice
+        case "$choice" in
+            y|Y ) 
+                if [[ -d "$1" ]]; then
+                    break
+                else
+                    printf "<FOLDER NOT FOUND> Type a valid path!\n"
+                fi
+            ;;
+            n|N ) read -p "The path to your Visual Paradigm (APP FOLDER) is: " $1 ;;
+            * ) printf "Invalid input\n";;
+        esac
+    done
+    app_dir="$1"
+}
+
+function read_Plugin_Path(){
```

Comment:
The same error happens with this function on Windows. Can you please change it?
```bash
function read_Plugin_Path(){
    plugin_path="$1"
    while true; do
        echo "Visual Paradigm Plugin Path: $plugin_path"
        read -p "Confirm (y/n)?: " choice
        case "$choice" in
            y|Y ) break;;
            n|N ) 
                echo "The path to your Visual Paradigm (PLUGIN FOLDER) is: "
                read -r plugin_path ;;
            * ) printf "Invalid input\n";;
        esac
    done
    plugin_dir="$plugin_path"
}
```

Reply:
Fixed on [1a15e8c](https://github.com/OntoUML/ontouml-vp-plugin/pull/146/commits/1a15e8c0dbadab4c77a18e0df57a28ff273ec29b)